error_test.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'environment'
+
+class ErrorTest
+  include Environment
+  attr_accessor :depot, :control, :board
+
+  def initialize
+    @board = {
+      players: {},
+      control: {},
+      status: nil
+    }
```

Comment:
I would define this kind of hashes using one line `@board = { players: {}, control: {}, status: nil }`